src/migrations/20200803230040-create-offers-approval-events-table.ts

```diff
@@ -0,0 +1,42 @@
+import { QueryInterface, UUID, DATE, ENUM } from "sequelize";
+import { approvalStatuses } from "../models/ApprovalStatus";
+
+export = {
+  up: (queryInterface: QueryInterface) =>
+    queryInterface.createTable(
+      "OfferApprovalEvents",
+      {
+        uuid: {
+          allowNull: false,
+          primaryKey: true,
+          type: UUID
+        },
+        adminUserUuid: {
+          allowNull: false,
+          references: { model: "Admins", key: "userUuid" },
+          onDelete: "CASCADE",
+          type: UUID
+        },
+        offerUuid: {
+          allowNull: false,
+          references: { model: "Offers", key: "uuid" },
+          onDelete: "CASCADE",
+          type: UUID
+        },
+        status: {
+          allowNull: false,
+          type: ENUM<string>({ values: approvalStatuses })
```

Comment:
Cambiaria el type por:
```
type: "approval_status"
```
Las migraciones de eventos quedaron medio legacy. Lo haria como en `src/migrations/20200606221857-add-approval-status-to-companies.ts`